backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlDatabase.java

```diff
@@ -324,9 +329,9 @@ public void setActiveUser(String username) {
         }
       }
     } else {
-      if (username == null && connectionProvider.getActiveUser() != null
-          || username != null && !username.equals(connectionProvider.getActiveUser())) {
+      if (!Objects.equals(username, connectionProvider.getActiveUser())) {
         clearCache();
+        listener.userChanged();
```

Comment:
why are both needed , i would expect the listener to be triggered when the user is changes , now its; clearCache() then clearCache()

Reply:
that is correct, will fix